src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+
+
+namespace System.Reflection.Emit.Experimental
+{
+    class AssemblyBuilder: System.Reflection.Assembly
```

Comment:
```suggestion
    public class AssemblyBuilder: System.Reflection.Assembly
```
The type needs to be public so that it can be called from other assemblies.